kickoff/jsonexportl10n.py

```diff
@@ -19,33 +20,59 @@
 def l10nExport(releaseName):
     # Export the l10n changeset for a product
     releaseTable = getReleaseTable(releaseName)
+
+    if releaseName.endswith(config.BETA_AGGREGATION_KEYWORD):
+        l10n_list = _aggregateBetaLocales(releaseTable, releaseName)
+    else:
+        l10n_list = _getLocalesByReleaseName(releaseTable, releaseName)
+
+    l10n_list["version"] = config.JSON_FORMAT_L10N_VERSION
+    return myjsonify(l10n_list)
+
+
+def _aggregateBetaLocales(releaseTable, releaseName):
+    # We need to keep the first "b", so we can look up names like "Firefox-32.0b"
+    beta_keyword_length = len(config.BETA_AGGREGATION_KEYWORD) - 1
+    betas_substring = releaseName[:-beta_keyword_length]
```

Comment:
My connection dropped whily I was boarding a plane. You should see them by now 😃